examples/thermostat/qpg/include/AppConfig.h

```diff
@@ -19,4 +19,6 @@
 #ifndef APP_CONFIG_H
 #define APP_CONFIG_H
 
+// Mapping of Functional buttons
```

Comment:
do we need this comment? It seems to not do anything...

Reply:
Will remove and use "#pragma once" as well.